cypress/e2e/registration.cy.js

```diff
@@ -1,11 +1,42 @@
 /// <reference types='cypress' />
 
+const { generateUser } = require('../support/generateUser');
 describe('Student Registration page', () => {
   before(() => {
-
+    cy.visit('https://demoqa.com/automation-practice-form');
   });
 
-  it('', () => {
+  it('should send the form with filled fields and check modal', () => {
+    const {
+      userName,
+      email,
+      mobile,
+      currentAddress,
+      state,
+      city
+    } = generateUser();
```

Comment:
The `generateUser` function also returns a `subject` field, but it is not used in the test. According to the task, all fields should be filled. Make sure to include the subject field in the form filling process.